tests/test_metric.py

```diff
@@ -125,32 +128,33 @@ def test_msssim(self, einstein_img, curie_img):
         curie_img.requires_grad_()
         assert po.metric.ms_ssim(einstein_img, curie_img).requires_grad
 
-    @pytest.mark.parametrize('func_name', ['noise', 'mse', 'ssim', 'ms-ssim'])
-    @pytest.mark.parametrize('size_A', [1, 3])
-    @pytest.mark.parametrize('size_B', [1, 2, 3])
+    @pytest.mark.parametrize('func_name', ['mse', 'ssim', 'ms-ssim', 'nlpd', 'nspd'])
```

Comment:
the test for `po.tools.add_nosie` was removed -- why?

Reply:
The reason I removed it was probably because it is a small function and the batch handling is simple (like as simple as addition), so the testing it together with `ssim`, `ms-ssim`, `nlpd` and `nspd` was not convenient. In fact I'm also going to remove `mse` from this test for convenience. So I think I can write another function to test for `noise` and `mse`.